Anexo de navbar

SilviaPabon opened the pull request:
-Chequear los comentarios de cada commit
-Es importante manejar unos comentarios sencillos pero que nos sirvan en un futuro si revisamos de nuevo el trabajo.

SilviaPabon committed:
Anexos en html

-Anexo de la barra de navegación
-Anexo de script para facilitar anexo de íconos

SilviaPabon committed:
Anexos en css

-Anexo de estilos para la barra de navegación tanto para web como para móvil:
*Botones
*Despliegue menú hamburguesa
*etc
-Anexo cambios en .hero height
-Cambios en .btn (descomentar transition)

SilviaPabon committed:
Anexos en index.js

-Se anexa código que permite que se pueda expandir o esconder la navbar de forma vertical, luego de dar clic en el botón hamburguesa (cuando se está en una resolución móvil)
-Se anexa código que facilita la accesibilidad en el botón de hamburguesa agregando un aria-label indicando "abrir menú" "cerrar menú".
-Pendiente anexar mejores comentarios.

SilviaPabon committed:
Anexo de aria-label en el html complementario al código realizado en index.js

Anexo de aria-label en el html, línea 30, complementario al código realizado en index.js para facilitar la accesibilidad

PedroChaparro commented on assets/styles/main.css:
```diff
@@ -151,4 +196,39 @@ html{
+        width:fit-content;
+    }
+
+    .nav__navigation{
```
En el .nav__navigation del media query se podría poner un gap pequeño para que los items se separen un poco.

PedroChaparro commented on assets/styles/main.css:
```diff
@@ -151,4 +196,39 @@ html{
+        left: 100%;
+        transition: left 0.3s;
+    }
+
```
Para el ancho fijo se podría usar vw en lugar de px (para que cambie según el ancho del dispositivo). Por ejemplo 80vw sería el 80% del ancho.

PedroChaparro reviewed (commented):
Ancho fijo y separación

SilviaPabon committed:
Cambios solicitados por @PedroChaparro

-Cambios en .nav__item
-Gap en .nav__navigation
-Anexo de comentarios en el media query

SilviaPabon commented:
@PedroChaparro Listos los cambios.

SilviaPabon committed:
Anexo fondo amarillo menú de hamburguesa

PedroChaparro committed:
Merge branch 'dev' into navbar
